shared/lib/accounts/solana-wallet-snap.ts

```diff
@@ -3,7 +3,8 @@ import { SnapId } from '@metamask/snaps-sdk';
 // the Snap is being pre-installed only for Flask build (for the moment).
 import SolanaWalletSnap from '@metamask/solana-wallet-snap/dist/preinstalled-snap.json';
 
-export const SOLANA_WALLET_SNAP_ID: SnapId = SolanaWalletSnap.snapId as SnapId;
+export const SOLANA_WALLET_SNAP_ID: SnapId =
+  'local:http://localhost:8080' as SnapId;
```

Comment:
This is needed for testing snap locally. Should be removed before merging.

Reply:
What URL should this be?  We could use a feature flag for dev mode vs prod.

Reply:
I'm actually not sure, I would assume that the snap is hosted somewhere. Maybe this can be added into an .env variable for now.

I'll get a snap engineer to review this as well.